### ISSUE TITLE
Allow customizer renaming functionality to be invoked automatically upon saving

### ISSUE THREAD

**Issue by allynt** [ES-DOC/esdoc-questionnaire#197]
This is discussed in #169.

There is code to propagate a customization's name change to all associated customizations recursively.  This gets called from the view.  However, if I ever change a customization's name outside of the view, this code will not be triggered. 

This ticket is to have that code run automatically, without compromising the view logic.

**Comment by murphysj** [comment]
Is this applicable any more since #169 was closed?

**Comment by allynt** [comment]
Yes.  This ticket refers to moving this functionality to the save method of the customization.  Currently it is part of the customize view.  This means if the customization name is ever changed outside of the Q front-end (either in code or via the Django Admin), those changes would not propagate.
